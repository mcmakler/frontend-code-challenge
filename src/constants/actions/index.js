import * as advertisementTypes from './advertisement-types';

export {
  advertisementTypes
};
