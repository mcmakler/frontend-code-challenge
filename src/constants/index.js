export default {
  currencyLabel: 'â‚¬',
  roomLabel: 'Zimmer',
  defaultWordCount: 10,
  titleWordCount: 7,
  advertisementType: {
    rent: 'mieten',
    sell: 'kaufen'
  },

  specialCharRegex: /[&/\\#+()$~%.'":*?<>{}]/g
};
