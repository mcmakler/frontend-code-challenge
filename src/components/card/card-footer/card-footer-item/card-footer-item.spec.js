import React from 'react';
import CardFooterItem from './card-footer-item';
import ReactTestRenderer from 'react-test-renderer';
import constants from '../../../../constants';

describe('CardFooterItem specs', () => {
  it('should match snapshot with data', () => {
    const component = ReactTestRenderer.create(
      <CardFooterItem value={111} prelabel='price' postlabel={constants.currencyLabe}/>
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
