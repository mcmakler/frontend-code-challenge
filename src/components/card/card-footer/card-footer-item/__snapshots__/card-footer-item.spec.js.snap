// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardFooterItem specs should match snapshot with data 1`] = `
<span
  className="cardFooter__item__text "
>
  price
   
  111
   
  
</span>
`;
